gtsfm/averaging/translation/averaging_1dsfm.py

```diff
@@ -209,7 +211,7 @@ def _sample_kde_directions(w_i2Ui1_measurements: BinaryMeasurementsUnit3, num_sa
     Returns:
         List of sampled Unit3 directions.
     """
-    w_i2Ui1_list = [w_i2Ui1.measured() for w_i2Ui1 in w_i2Ui1_measurements]
+    w_i2Ui1_list = [w_i2Ui1_measurements[idx].measured() for idx in range(len(w_i2Ui1_measurements))]
```

Comment:
@ayushbaid @akshay-krishnan the change in this line seems less readable now

Reply:
Reverting.